pkg/controller/controller.go

```diff
@@ -197,8 +195,21 @@ func (c *Controller) runController() {
 	runTimer, timerDone := inctimer.New()
 	defer timerDone()
 
+	var (
+		retryReasonControllerUpdated   = "controllerUpdated"
+		retryReasonControllerTriggered = "controllerTriggered"
+		retryReasonTimerExpired        = "timerExpired"
+	)
```

Comment:
Nit: better declare these as constants and move them at line 23 with all the others.